src/components/Search.js

```diff
@@ -0,0 +1,36 @@
+import styled from "styled-components";
+
+function Search(){
+    return(
+    
+        <SearchForm>
+            <label name="SearchField" for="searchField">Search for Food:</label>
+            <input id="searchField" name="SearchField" />
+        </SearchForm>
+    
+)
+}
+
+export default Search
+
+const SearchForm = styled.form`
+
+display: flex;
+flex-wrap: wrap;
+max-width: 70%;
+justify-content: center;
+gap: 20px;
+margin-top: 1em;
+
+label {
+    font-weight: 900;
+}
+
+#searchField {
```

Comment:
Please don't use the `id` selector. Maybe add a new styled-component with the name `SarchField`?